packages/config/src/chains.ts

```diff
@@ -1307,6 +1312,22 @@ export const uniqueAlpha = new Parachain({
   ws: 'wss://unique-alpha.unique.network',
 });
 
+export const zeitgeist = new Parachain({
+  assetsData: [
```

Comment:
I don't think that you need this at all.

Reply:
You're right, it's not necessary.
I've removed it and updated the dev versions